networking/aws_load_balancer_operator/add-tls-termination-and-ddos-protection.adoc

```diff
@@ -0,0 +1,11 @@
+:_content-type: ASSEMBLY
+[id="tls-termination-and-ddos-protection"]
+= Adding TLS termination and DDos protection
+include::modules/common-attributes.adoc[]
+:context: tls-termination-and-ddos-protection
+
+toc::[]
+
+You can add TLS termination and DDos protection on the load balancer.
+
+include::modules/adding-tls-termination-and-ddos-protection.adoc[leveloffset=+1]
```

Comment:
Can we drop the DDoS protection from this page. The DDoS protection is present but cannot be tested. So including it in the documentation right now is not the right move.

Reply:
Sure- dropped the module from this assembly file.